lib/alice/console.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Alice.Console do
+  @moduledoc """
+  Starts up the Console to allow testing of Alice's responses
+  """
+  @doc "Boots up the console and all associated constructs"
+  def start() do
+    setup_alice()
+    IO.puts("Starting Alice Console")
+    repl()
+  end
+
+  defp repl(last_message \\ "")
+  defp repl("exit"), do: IO.puts("Goodbye!")
+
+  defp repl(_last_message) do
+    message =
+      IO.gets("alice> ")
+      |> String.trim("\n")
+      |> String.replace("@alice", "<@alice>")
+
+    Alice.HandlerCase.send_message(message)
+    :timer.sleep(1)
+    repl(message)
+  end
+
+  defp setup_alice() do
+    Application.ensure_all_started(:alice)
+    Application.put_env(:alice, :chat_backend, :console)
+    Application.put_env(:alice, :outbound_client, Alice.ChatBackends.ConsoleOutbound)
+    Alice.start(:normal, %{})
+    Logger.configure(level: :error)
```

Comment:
I think this should respect the log level configured. The new installer, sets it to `:warn` in the [config template](https://github.com/alice-bot/alice_new/blob/master/templates/new_handler/config/config.exs.eex)

Reply:
So the reason the console is overriding the log level is because existing handlers and bots mostly appear to have a log level of `:info`. This causes the routing log to pop up in the middle of the console, which looks weird. The usage of the console is also just different then when you're actually running the application, so it makes me sense to me that we're overriding it if it's not at least set to `:warn`.

Reply:
It just feels kind of like being a bad citizen. Newly generated handlers start their loggers and are defaulted to `:warn`. Having said that, I do get the argument that this is a console and should have clean output. On the other hand, I could see a use case for wanting to log while in the console for debugging.

What about configuring the Logger's output format to look better n the console?

Reply:
<img width="808" alt="Screen Shot 2020-04-15 at 2 29 14 PM" src="https://user-images.githubusercontent.com/667667/79390977-8596ee00-7f25-11ea-9da5-de36a4119e2b.png">
This is what it looks like without overriding the logger's existing log level. We could set it to `:warn`, and if they want to debug they can warn. Like I said, the console just seems like a different use case to me. You're not debugging your bot's internals, it's a console to have a conversation with the bot without an adapter. Hell, the thing this is meant to replace is having to use the slack adapter entirely (where you wouldn't get the logs coming through either).

Perhaps we set a new configuration variable for `:console_log_level`? So that people can override it if they wish?